File: 100 problem/4find_largest_element.py
# // Write a program to find the largest element in an array.

# method 1
def find_largest_element(arr):
    return max(arr)

# method 2
def find_largest_element1(arr):
    maxi = arr[0]
    i = 1
    arr_size = len(arr)
    while i < arr_size:
        if maxi < arr[i]:
            maxi = arr[i]
        i += 1
    return maxi

arr = [10, 20, 40, 1, 2, 0, 500, 24, 100]
print(find_largest_element1(arr))

File: 100 problem/4find_largest_element.js
// Write a program to find the largest element in an array.

function find_largest_element(arr) {
    let i = 1;
    let max = arr[0];
    let arr_size = arr.length;

    while (i < arr_size) {
        if(max < arr[i]) max = arr[i];
        i++;
    }

    return max;
}

arr = [10, 20, 40, 1, 2, 0, 500, 24, 100]

maxi = Math.max(...arr)
console.log(maxi)

console.log(find_largest_element(arr))